doc/cookbook/source/references.bib

```diff
@@ -59,4 +59,13 @@ @book{scholkopf2002learning
   author={B. Sch{\"o}lkopf and A.J. Smola},
   year={2002},
   publisher={MIT press}
-}
\ No newline at end of file
+}
+
+@inproceedings{gao2011discriminative,
```

Comment:
you mean the publication or the format..? 
The format is from the BibTeX citing of google scholar..
and publication is in relaxedtree.h